packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -621,7 +622,47 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         containerScope: FluidObject = context.scope,
         existing?: boolean,
     ): Promise<ContainerRuntime> {
-        // If taggedLogger exists, use it. Otherwise, wrap the vanilla logger:
+        let existingFlag = true;
+        if (!existing) {
+            existingFlag = false;
+        }
+        return this.loadRuntime({
+            context,
+            registryEntries,
+            existing: existingFlag,
+            requestHandler,
+            runtimeOptions,
+            containerScope,
+        });
+    }
+
+    /**
+     * Load the stores from a snapshot and returns the runtime.
+     * @param context - Context of the container.
```

Comment:
These comments should probably be changed to reflect the single param with properties.  Also, containerScope is missing. (it's also missing from the doc comment on `load`)